static/js/components/ShowSummaries.jsx

```diff
@@ -36,11 +36,22 @@ export const useStyles = makeStyles((theme) => ({
   },
 }));
 
-const ShowSummaries = ({ summaries = [] }) => {
+const ShowSummaries = ({ summaries = [], showAISummaries = true }) => {
   const styles = useStyles();
   const renderCommentText = () => {
-    if (summaries?.length > 0) {
-      return summaries[0].summary;
+    let filteredSummaries = [...summaries].filter(
+      (summary) =>
+        summary?.summary &&
```

Comment:
our is-null test comes up again!

Reply:
I know 😓. I promise I will have another huge refactor PR dedicated to this 😉